scene/2d/physics_body_2d.cpp

```diff
@@ -594,7 +594,6 @@ real_t RigidBody2D::get_mass() const {
 
 void RigidBody2D::set_inertia(real_t p_inertia) {
 
-	ERR_FAIL_COND(p_inertia <= 0);
```

Comment:
Does it make sense to set a negative inertia?

Reply:
According to this https://docs.godotengine.org/en/3.1/classes/class_rigidbody2d.html#class-rigidbody2d-property-inertia , a negative inertia means erasing user custom inertia and recalculate it.
Maybe we should use another param like `custom_inertia` to control it? If so, we can keep in same with setting custom gravity center in the future, since you can't set a "negative gravity center".

Reply:
I think the "(or negative)" part of the documentation should simply be removed, setting it to 0 is sufficient for a way to erase the custom inertia.

Reply:
So how about adding a bool param `custom_inertia` to control it ?
If we add it, this will be same mechanism with `custom_gravity_center`(future feature) , 
you can't use a zero value meaning erase custom gravity center(zero means a custom gravity center at origin position).

Reply:
Well I prefer not future proofing the API, we can do these changes if we do add a way to override the gravity center. Until this, it's better to keep the API unchanged in this PR and just fix the issue.